src/Azure.Functions.Cli/Actions/LocalActions/InstallExtensionAction.cs

```diff
@@ -85,39 +85,42 @@ public async override Task RunAsync()
                 return;
             }
 
-            if (CommandChecker.CommandExists("dotnet"))
+            if (!string.IsNullOrEmpty(ConfigPath) && !FileSystemHelpers.DirectoryExists(ConfigPath))
             {
-                if (!string.IsNullOrEmpty(ConfigPath) && !FileSystemHelpers.DirectoryExists(ConfigPath))
-                {
-                    throw new CliArgumentsException("Invalid config path, please verify directory exists");
-                }
+                throw new CliArgumentsException("Invalid config path, please verify directory exists");
+            }
 
-                var extensionsProj = await ExtensionsHelper.EnsureExtensionsProjectExistsAsync(_secretsManager, Csx, ConfigPath);
+            var extensionsProj = await ExtensionsHelper.EnsureExtensionsProjectExistsAsync(_secretsManager, Csx, ConfigPath);
```

Comment:
This will add an extension csproj to the project. I think the correct fix is to not add this at all if there are no extensions to be installed.